src/app/pages/client-reviews/client-reviews.component.spec.ts

```diff
@@ -0,0 +1,94 @@
+import { mockReviewsResponse } from 'src/app/shared/mocks';
+import { ClientReviewsService } from './../../services/client-reviews/client-reviews.service';
```

Comment:
@dorothyas great work here, kindly use absolute imports to import the ClientReviewService

Reply:
done